### ISSUE TITLE
Idea for improvement: Showing LCA scenario based results as line graphs over time

### ISSUE THREAD

**Issue by CHarpprecht** [LCA-ActivityBrowser/activity-browser#417]
Hi,
I have a suggestion for the graphs showing LCA results for scenario based LCA:
It would be nice to not only have bar graphs, but also line graphs to illustrate the LCA results over time for different functional units per impact category.

For example, like in this graph
<img width="372" alt="line graphs" src="https://user-images.githubusercontent.com/47253039/85424245-f7b82e80-b577-11ea-8868-d6858349da0f.png">

**Comment by bsteubing** [comment]
@CHarpprecht interesting idea, thanks. Right now I am a bit hesitant as we provide general functionality to handle scenarios (or even combination of scenarios), but by default those don't have to be scenarios over time. So I am not sure how to implement this at this point. Ideas are welcome of course!

**Comment by marc-vdm** [comment]
Potential solution:
Could we not just add this as a (radio button) option?
Like:
O Scenario bars O Scenario line

**Comment by CHarpprecht** [comment]
I am also not sure, how to implement this in the end. 
Probably this isn't straight-forward given how the labelling of scenarios work currently. I assume, the scenario label is just a string right now, so it might require some extra effort to get to a nice line graph with a correct time axis on the x-axis?

**Comment by marc-vdm** [comment]
Fair point about the scenario labelling, I had forgotten that. Sorry about getting your hopes up.
Perhaps this could become a todo if/when we get scenario names.

@dgdekoning do you know the current status of AB understanding scenario names?
